api/v1beta1/glance_webhook.go

```diff
@@ -441,3 +457,15 @@ func GetCrMaxLengthCorrection(name string, apiType string) int {
 
 	return (defaultCrMaxLengthCorrection + len(name) + len(apiType) + 2)
 }
+
+// ValidateTopologyNamespace - returns a field.Error when Glance / GlanceAPI
+// references a Topoology deployed on a different namespace
+func ValidateTopologyNamespace(topology *TopologyRef, basePath field.Path, ns string) (*field.Error) {
```

Comment:
should this also go into the infra-operator so that we don't have to copy it to all operators?

Reply:
This is an interesting perspective. Definitely we can move the content of the function in infra-operator and provide a basic validation that can be called from other operators. I'll try that (probably tomorrow).